pycbc/mchirp_area_improved.py

```diff
@@ -67,8 +72,27 @@ def from_cli(args):
                                  'b0': args.src_class_lum_distance_to_delta[0],
                                  'b1': args.src_class_lum_distance_to_delta[1],
                                  'm0': args.src_class_mchirp_to_delta},
-            'mass_gap': args.src_class_mass_gap_separate}
-
+            'mass_gap': args.src_class_mass_gap_separate,
+            'lal_cosmology': args.src_class_lal_cosmology}
+
+def redshift_estimation(distance, distance_std, lal_cosmology):
+    """Takes values of distance and its uncertainty and returns a
+       dictionary with estimates of the redshift and its uncertainty.
+       If the argument 'lal_cosmology' is True, it uses Planck15 cosmology
+       model as defined in lalsuite instead of the astropy default.
+    """
+    if lal_cosmology:
```

Comment:
can you add a comment pointing to the line of code defining this in pesummary? 
I think it is this currently 
https://git.ligo.org/lscsoft/pesummary/-/blob/master/pesummary/gw/cosmology.py#L43
& you should mention the function name Planck15_lal_cosmology()